sale_payment_method_auto_pay_selection/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+=====================================================
+Sale Payment Method - Selection for Automatic Payment
+=====================================================
+
+* Adds an selection to payment methods to configure when an automatic payment is allowed to be created.
```

Comment:
It seems this description is slightly misleading. Is this not more about being able to mark payment methods not to allow automatic payment?